modules/eco-node-health-check-operator-about.adoc

```diff
@@ -58,7 +58,40 @@ During the upgrade process, nodes in the cluster might become temporarily unavai
 [id="understanding-nhc-operator-workflow_{context}"]
 == Understanding the Node Health Check Operator workflow
 
-When a node is identified as unhealthy, the Operator checks how many other nodes are unhealthy. If the number of healthy nodes exceeds the amount that is specified in the `minHealthy` field of the `NodeHealthCheck` CR, the controller creates a remediation CR from the details that are provided in the external remediation template by the remediation provider. After remediation, the node's health status is updated accordingly.
+When a node is identified as unhealthy, the Node Health Check Operator checks how many other nodes are unhealthy. If the number of healthy nodes exceeds the amount that is specified in the `minHealthy` field of the `NodeHealthCheck` CR, the controller creates a remediation CR from the details that are provided in the external remediation template by the remediation provider. After remediation, the kubelet updates the node's health status.
 
-When the node turns healthy, the controller deletes the external remediation template
-and updates the node's health status.
\ No newline at end of file
+When the node turns healthy, the controller deletes the external remediation template.
+
+[id="preventing-nhc-conflict-with-mhc_{context}"]
+== Preventing node health check conflicts with machine health check
```

Comment:
I wonder if we should rewrite this. _Preventing_ makes it sound like something the user needs to do, rather than what OCP does. Does this sound OK? (It's accurate but I'm not sure it sounds OK for a heading.)

About how {product-title} prevents node health check and machine health check conflicts

Reply:
@mburke5678 I'm not sure if it would be technically correct to rewrite it as you have suggested. Before an internal peer review comment, my original heading was "Understanding node health check behavior with machine health check".

Would it be ok to rewrite it as _About how node health checks prevent conflicts with machine health checks_ ?

Reply:
@abhatt-rh That sounds good.